docs/releasing.md

```diff
@@ -12,8 +12,8 @@
       * Accumulation of many changes, none of which are individually big enough to warrant a minor bump, but which together are. This does not include changes which are purely internal to the extension, such as refactoring, or which are only available behind a feature flag.
 1. Double-check that the node version we're using matches the one used for VS Code. If it doesn't, you will then need to update the node version in the following files:
     * `.nvmrc` - this will enable `nvm` to automatically switch to the correct node version when you're in the project folder
-    * `.github/workflows/main.yml` - all the "node-version: <version>" settings
-    * `.github/workflows/release.yml` - the "node-version: <version>" setting
+    * `.github/workflows/main.yml` - all the "node-version: '[VERSION]'" settings
+    * `.github/workflows/release.yml` - the "node-version: '[VERSION]'" setting
```

Comment:
Markdown was considering `<version>` as HTML. This means that it triggered a warning in markdownlint, and didn't show up in [the rendered version](https://github.com/github/vscode-codeql/blob/9015a57732d332e59ccdf961fdaa380c1348b931/docs/releasing.md) either.